chore:  updated readme for rn >=0.60, removed old typo info (#278)
callstack/react-native-fbads: README.md

  - [Run it on iOS / Android](#3-run-it-on-ios--android)
- [Credits](#credits)

## Prerequisites

You must have Facebook developer account in order to start integrating your app with this library. If you don't have one sign up [here](https://developers.facebook.com/).

Follow the instructions on [react-native-fbsdk](https://github.com/facebook/react-native-fbsdk) to integrate the **Facebook SDK** into your project.

## Installation

> Add the package to your project using your favorite package manager yarn:

Reviewer: ```suggestion
Add the package to your project using either yarn:
```

```bash

> $ yarn add react-native-fbads

Reviewer: Remove dollars, it's not beginners friendly and it makes it harder to copy whole thing easily
```suggestion
yarn add react-native-fbads
```

```

or npm:

```bash

> $ npm install --save-dev react-native-fbads

Reviewer: ```suggestion
npm install --save-dev react-native-fbads
```

> npm install --save-dev react-native-fbads

Reviewer: ```suggestion
npm install --save react-native-fbads
```

```

#### For React Native >= 0.60

If you are working with RN >= 0.60 automatic linking should do the job for you.

<details>
<summary>For React-Native < 0.60</summary>
Link the native dependencies:

```bash
$ react-native link react-native-fbads
```

</details>

> ### 1. Get a Placement ID

Reviewer: ```suggestion
### Get a Placement ID
```

Follow [Facebook's instructions](https://www.facebook.com/help/publisher/1195459597167215) to create placement IDs for your ads.

You can skip the _Integrate the SDK_ step of that guide, as you've already integrated the Facebook SDK in previous steps.

> ### 2. Testing integration on simulators/devices

Reviewer: ```suggestion
### Testing integration on simulators/devices
```

  - [Run it on iOS / Android](#3-run-it-on-ios--android)
- [Credits](#credits)

## Prerequisites

You must have Facebook developer account in order to start integrating your app with this library. If you don't have one just sign in [here](https://developers.facebook.com/).

Follow the instructions on [react-native-fbsdk](https://github.com/facebook/react-native-fbsdk) to integrate the **Facebook SDK** into your project.

> Note that for iOS, it's [recommended you use Cocoapods](https://developers.facebook.com/docs/ios/getting-started/advanced) rather than the manual installation.

Reviewer: Let's remove this entirely, they have up-to-date autolinking instructions 
```suggestion
```

$ yarn add react-native-fbads
```

### For RN < 0.60

Link the native dependencies

```bash
$ react-native link react-native-fbads
```

#### iOS

If you have more than one Targets on your Xcode project, you might link some of them manually by dragging `Libraries/ReactNativeAdsFacebook.xcodeproj/Products/libReactNativeAdsFacebook.a` to 'Build Phases' -> 'Link Binary With Libraries'.

#### For RN >= 0.60

> If you are working with RN >= 0.60 automatic linking should do the job for you.

Reviewer: Let's remove this whole section
